### ISSUE TITLE
Don't use an ARIA progressbar to indicate steps

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#171]
### Problem

[molecules/molecules.progress](https://a-ui.github.io/core_branding_scss/#molecules-progress)

#### Example 1

1. Although it's theoretically valid to use the ARIA role progressbar to indicate steps, we advise against it. The current step is stated in text (step 1 of 5), therefore it's redundant to indicate this for a second time using ARIA roles and attributes.
2. The label element can only be used in combination with an input element.

```html
<div class="m-progress">
  <label class="m-progress__label">Stap 1 van 5</label>
  <div class="m-progress__inner">
    <div class="m-progress__bar" role="progressbar" aria-valuenow="20" aria-valuemin="0" aria-valuemax="100" style="width: 20%"></div>
  </div>
</div>
```

#### Example 2,3 and 4

* The label element can only be used in combination with an input element.

#### Example 1

1. Delete the ARIA role and attributes
2. Use a heading instead of a label element

Suggestion:

```html
<div class="m-progress">
  <h2>Stap 1 van 5</h2>
  <div class="m-progress__inner">
    <div class="m-progress__bar" style="width: 20%"></div>
  </div>
</div>
```

#### Example 2,3 and 4

* Use a div instead of a label element

```html
<div class="m-progress m-progress--inline">
  <div class="m-progress__inner">
    <div class="m-progress__bar" role="progressbar" aria-valuenow="20" aria-valuemin="0" aria-valuemax="100" style="width: 20%"></div>
  </div>
  <div class="m-progress__label">20%</div>
</div>
```

### Articles and links

[WAI-ARIA 1.2: progressbar (role)](https://www.w3.org/TR/wai-aria-1.2/#progressbar)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)

**Comment by Veyfeyken** [comment]
Example 1: the ARIA attributes are still present.
Example 2, 3 and 4: role="progressbar" is missing.
